src/sidebar/components/spinner.js

```diff
@@ -1,20 +1,25 @@
 'use strict';
 
-// @ngInject
-function SpinnerController($animate, $element) {
-  // ngAnimate conflicts with the spinners own CSS
-  $animate.enabled(false, $element);
-}
+const { createElement } = require('preact');
 
-module.exports = {
-  controller: SpinnerController,
-  controllerAs: 'vm',
-  template: `
-    <div class="spinner__container">
-      <span class="spinner">
-        <span><span>
-        </span></span>
+/**
+ * Loading indicator.
+ */
+function Spinner() {
+  // The `spinner__container` div only exists to center the spinner within
+  // the `<spinner>` Angular component element. Once consumers of this component
+  // have been converted to Preact, we should be able to remove this.
+  return (
+    <div className="spinner__container">
+      <span className="spinner">
+        <span>
```

Comment:
What do all of these `span`s do? :)

Reply:
I added a comment to refer the user to the CSS